tests.cfg

```diff
@@ -33,6 +33,7 @@ test-eggs =
     plone.app.redirector
     plone.app.registry
     plone.app.relationfield
+    plone.app.robotframework [test]
```

Comment:
@datakurre I've been banging my head against the wall on this... but might it be that the robot tests on `plone.app.robotframework` are already broken? For example on `p.a.rf/tests/test_autologin_library.robot` the keyword `I open a personal menu` tries to get `#portal-personaltools > a` ([source](https://github.com/plone/plone.app.robotframework/blob/master/src/plone/app/robotframework/tests/test_autologin_library.robot#L53)) and that is nowhere to be found (tested it debugging it, but also on demo.plone.org)

If we do not run the tests for plone.app.robotframework, but keep its changes, then we can test for example my port on [`plone.app.discussion`](https://github.com/plone/plone.app.discussion/pull/214) and use that for testing the GHA and Jenkins integration of playwright, which I have sort of working, but need to have stable tests to run them with 😄